### PR TITLE
Feature 13412 - Support for SBC-Staff

### PR DESCRIPTION
**Pull request by ritvick**
*Issue #:* bcgov/entity/issues/13412

*Description of changes:*
Support for SBC-Staff

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/store/getters/state-getters.ts [hunk 1]
```diff
@@ -100,11 +100,9 @@ export const isGovAccountUser = (state: StateIF): boolean => {
 
 /** Whether the user is SBC Staff (which is not the same as Staff). */
 export const isSbcStaff = (state: StateIF): boolean => {
-  if (isGovAccountUser(state)) {
-    const orgInfo = getOrgInformation(state)
-    return (orgInfo?.branchName?.includes('Service BC') || false)
-  }
-  return false
+  const orgInfo = getOrgInformation(state)
+  console.log(orgInfo)
```
Comment (seeker25) [inline src/store/getters/state-getters.ts:104]: console.log needs to be removed

File: src/store/getters/state-getters.ts [hunk 2]
```diff
@@ -100,11 +100,8 @@ export const isGovAccountUser = (state: StateIF): boolean => {
 
 /** Whether the user is SBC Staff (which is not the same as Staff). */
 export const isSbcStaff = (state: StateIF): boolean => {
-  if (isGovAccountUser(state)) {
-    const orgInfo = getOrgInformation(state)
-    return (orgInfo?.branchName?.includes('Service BC') || false)
-  }
-  return false
+  const orgInfo = getOrgInformation(state)
+  return (orgInfo?.orgType?.includes('SBC_STAFF') || false)
```
Comment (severinbeauvais) [inline src/store/getters/state-getters.ts:104]: Please update `org-information-interface.ts` with "orgType". Is it an array of strings?

Also, is orgType really optional? If not then please remove `?`.

Comment (severinbeauvais) [reply src/store/getters/state-getters.ts:104]: Oh sorry, I see you already added it to the interface!

Comment (severinbeauvais) [reply src/store/getters/state-getters.ts:104]: So it's a string of tokens?

Comment (ritvick) [reply src/store/getters/state-getters.ts]: Updated to string comparision

Comment (severinbeauvais) [reply src/store/getters/state-getters.ts]: Oh, OK, that's clear to me now! Thanks.

**Review by seeker25 — APPROVED** [review]

**Review by severinbeauvais — APPROVED** [review]
Looks good to me.

**Pull request merged**